### PR TITLE
feat(dns): remove hint about real target with dot

### PR DESCRIPTION
**Pull request by blemoineovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-12862
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description
remove help text about the real target having a dot when we do not enter the final dot in a new DNS record

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html [hunk 1]
```diff
@@ -34,26 +34,6 @@
                     ></span>
                 </span>
             </div>
-            <small
-                class="help-block"
-                data-ng-if="ctrl.targetIsRelativeDomain()"
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html:39]: is ctrl.targetIsRelativeDomain() still in use?
Same question for the key "domain_configuration_dns_entry_relative_warning_title" and "domain_configuration_dns_entry_relative_warning_tooltip".

File: packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html [hunk 2]
```diff
@@ -34,26 +34,6 @@
...
-            >
-                <span
-                    data-ng-bind="('domain_configuration_dns_entry_relative_warning_title' | translate) + ' &laquo;&nbsp;' + ctrl.model.target.target + '.' + ctrl.domain.displayName + '.&nbsp;&raquo;'"
-                ></span>
-                <button
-                    class="btn btn-icon"
-                    type="button"
-                    data-ng-click="ctrl.switchTargetToAbsolute()"
```
Comment (qpavy) [inline packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html:47]: Also clean the controller if the function is not use anywhere else

File: packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html (hunk 1 above)
Comment (qpavy) [inline packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html:39]: Also clean the controller if the function is not use anywhere else

**Review by qpavy — CHANGES_REQUESTED** [review]

File: packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html (hunk 1 above)
Comment (blemoineovh) [reply packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html:39]: yes, these functions and translations are used in other templates like MX and SRV

File: packages/manager/apps/web/client/app/domain/zone/record/CNAME/domain-zone-record-CNAME.html [hunk 3]
```diff
@@ -34,14 +34,6 @@
                     ></span>
                 </span>
             </div>
-            <small
-                class="help-block"
-                data-ng-if="ctrl.targetIsRelativeDomain()"
-            >
-                <span
-                    data-ng-bind="('domain_configuration_dns_entry_relative_warning_title' | translate) + ' &laquo;&nbsp;' + ctrl.model.target.target + '.' + '&nbsp;&raquo;'"
```
[the remark this answers is not in the document]
Comment (blemoineovh) [reply packages/manager/apps/web/client/app/domain/zone/record/CNAME/domain-zone-record-CNAME.html:42]: this is used in NS and SRV templates, i do not think the ticket said anything about these

[the remark this answers is not in the document]
Comment (blemoineovh) [reply packages/manager/apps/web/client/app/domain/zone/record/CNAME/domain-zone-record-CNAME.html:42]: ok i updated, the tickets says to change things in every place saying that the target is not ending with a dot

File: packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html (hunk 1 above)
Comment (blemoineovh) [reply packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html:39]: i removed these usages too

Comment (blemoineovh) [reply packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html:39]: removed switchTargetToAbsolute definition and usage in DNAME too

**Review by qpavy — DISMISSED** [review]

**Review by qpavy — APPROVED** [review]

File: packages/manager/apps/web/client/app/domain/dashboard/translations/Messages_fr_CA.json
Comment (anooparveti) [inline packages/manager/apps/web/client/app/domain/dashboard/translations/Messages_fr_CA.json:1]: Can not modify this file. only `Messages_fr_FR.json` is allowed.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/web/client/app/domain/dashboard/translations/Messages_fr_CA.json
Comment (blemoineovh) [reply packages/manager/apps/web/client/app/domain/dashboard/translations/Messages_fr_CA.json:1]: ok removed the changes in CA file

**Review by qpavy — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**